### PR TITLE
Tweak wording, mention NuGet options

### PR DESCRIPTION
**Pull request by mlimber**
I'd be fine leaving off the NuGet line in the table.

### CODE REVIEW

File: docs/framework/wpf/advanced/weak-event-patterns.md [hunk 1]
```diff
@@ -27,10 +27,11 @@ In applications, it is possible that handlers that are attached to event sources
...
 |Use a generic weak event manager class|Use a generic <xref:System.Windows.WeakEventManager%602> when an existing <xref:System.Windows.WeakEventManager> is not available, you want an easy way to implement, and you are not concerned about efficiency. The generic <xref:System.Windows.WeakEventManager%602> is less efficient than an existing or custom weak event manager. For example, the generic class does more reflection to discover the event given the event's name. Also, the code to register the event by using the generic <xref:System.Windows.WeakEventManager%602> is more verbose than using an existing or custom <xref:System.Windows.WeakEventManager>.|  
 |Create a custom weak event manager class|Create a custom <xref:System.Windows.WeakEventManager> when an existing <xref:System.Windows.WeakEventManager> is not available and you want the best efficiency. Using a custom <xref:System.Windows.WeakEventManager> to subscribe to an event will be more efficient, but you do incur the cost of writing more code at the beginning.|  
-  
+|Use a third-party weak event manager|NuGet has several weak event managers.|
```
Comment (BillWagner) [inline docs/framework/wpf/advanced/weak-event-patterns.md]: I like this addition.  What do you think about making this a link to NuGet.org, with appropriate search terms?

**Review by BillWagner — COMMENTED** [review]
I like these changes @mlimber 

I left a comment about the recommendation to explore NuGet packages. Do you think a link or search would be warranted here?

Let me know your thoughts, or make an update, and I'll :shipit: when we've made a decision on this idea.

File: docs/framework/wpf/advanced/weak-event-patterns.md (hunk 1 above)
Comment (mlimber) [reply docs/framework/wpf/advanced/weak-event-patterns.md]: That's fine by me, but I didn't know if that was legit in official documentation. Here's one such [search query](https://www.nuget.org/packages?q=weak+event+manager&prerel=false), and here's [one specific example](https://www.nuget.org/packages/ThomasLevesque.WeakEvent/). Various frameworks also offer this functionality (e.g., [Prism's](https://github.com/PrismLibrary/Prism-Documentation/blob/master/docs/wpf/Communication.md)). What would you prefer?

Comment (mlimber) [reply docs/framework/wpf/advanced/weak-event-patterns.md]: I added the NuGet link and mentioned Prism, too. See what you think.

**Review by BillWagner — APPROVED** [review]
Thanks for making the updates @mlimber 

I apologize for the lengthy delay. This PR dropped off my radar during the holiday break.

I've approved the changes, and I'll :shipit: now.

Thanks for contributing to docs.

**Pull request merged**